tests/test_converter.py

```diff
@@ -164,6 +165,37 @@ def test_encode_search_attribute_values():
         temporalio.converter.encode_search_attribute_values(["foo", 123])
 
 
+def test_decode_search_attributes():
```

Comment:
Is it possible to also add an integration test that sets a datetime attribute in a workflow? Or maybe alter `test_workflow_search_attributes` so that it will break without your fix?

Reply:
Leave it with me